src/Storefront/Framework/Twig/TemplateDataExtension.php

```diff
@@ -93,7 +94,23 @@ public function getGlobals(): array
         ];
     }
 
+    /**
+     * @deprecated tag:v6.3.0 - Use getStorefrontThemeConfig instead
```

Comment:
This should be `tag:v6.4.0`. The `v6.2.0` will already be release when this gets merged, so we need to extend that.